diesel/src/expression/expression_methods/sort_expression_methods.rs

```diff
@@ -0,0 +1,44 @@
+use expression::predicates::*;
+
+#[cfg(feature = "postgres")]
+pub trait Ordering : Sized {
```

Comment:
Ah, so maybe I should put these methods in that trait instead?  Hm.  On second thought, I guess not, since that trait is implemented for column types, and not for `Asc` and `Desc`.  So if I put `nulls_last()` there it would be possible to `order(foo.nulls_last())` but not `order(foo.desc().nulls_last())`, and that would make no sense at all ...